doc/getting-started/apl-nuc.rst

```diff
@@ -398,3 +401,53 @@ Set up Reference UOS
       :align: center
       :name: gsg-successful-boot
 
+
+.. _quick-setup-guide:
+
+Use the script to set up ACRN automatically
+===========================================
+
+It is little complicate to setup the SOS or UOS, so we provide a script to do it quickly and automatically.
+You can find the script `here
+<https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/getting-started/gsg_quick_setup.sh>`__
+and please note that should be run with root privilege since it will modify various system parameters.
+
+#. Installing Clear Linux and login system
+
+#. Open a terminal
+
+#. Run ``cd ~ && wget https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/getting-started/gsg_quick_setup.sh``
+
+#. Run ``sudo sh gsg_quick_setup.sh -s 28920`` to setup SOS
+
+   .. figure:: images/quick-setup-start.png
```

Comment:
Could you replace this picture by real text (and use `code-block:: console` to make it look like output on a terminal? (that makes it easier to update it later to newer versions, etc.)